tests/tests_suite_test.go

```diff
@@ -90,9 +90,9 @@ func TestTests(t *testing.T) {
 	RunSpecs(t, "Tests Suite")
 }
 
-var _ = SynchronizedBeforeSuite(testsuite.SynchronizedBeforeTestSetup, testsuite.BeforeTestSuitSetup)
+var _ = SynchronizedBeforeSuite(testsuite.SynchronizedBeforeTestSetup, testsuite.BeforeTestSuiteSetup)
```

Comment:
Please remove #1 from the commit. (It is keyword)

Reply:
Done